packages/contractkit/src/utils/signing-utils.ts

```diff
@@ -205,3 +205,12 @@ export function verifyEIP712TypedDataSigner(
   const valid = verifySignature(ensureLeading0x(trimmedData), signedData, expectedAddress)
   return valid
 }
+
+export function decodeSig(sig: any) {
```

Comment:
Just wondering if any of this is duping what's already in `https://github.com/celo-org/celo-monorepo/blob/master/packages/contractkit/src/utils/signing-utils.ts`?